### ISSUE TITLE
소문 듣고 찾아왔습니다 라면땅 시롸?

### ISSUE THREAD

**Issue by shan2366** [saewonlee/wordchain#1]
소문 듣고 찾아왔습니다 라면땅 시롸?

**Comment by saewonlee** [comment]
아니 shan선생님... 이거 읽어보세요 ㅠㅠ https://namu.wiki/w/%EB%9D%BC%EB%A9%B4%EB%95%85

**Comment by saewonlee** [comment]
> 소문 듣고 찾아왔습니다 라면땅 시롸?

아니 shan선생님... 이거 읽어보세요 ㅠㅠ https://namu.wiki/w/%EB%9D%BC%EB%A9%B4%EB%95%85
